docs/IoT.js-API-File-System.md

```diff
@@ -91,6 +110,20 @@ Closes the file of fd asynchronously.
 
 Closes the file of fd synchronously.
 
+#### fs.unlink(path, callback)
+
+* `path: String` - file path to be removed
+* `callback: Function(err) `
+ * `err: Error`
```

Comment:
Above Left space has meaning. `err` is the sub item of `callback`. Without left space, `err` will be shown in the same level of `callback`. Please see current document in IoT.js-API-File-System.md. There are many usages of this pattern for describing callback's parameters.